src/modules/orders/selectors/filled-orders.js

```diff
@@ -54,7 +64,7 @@ function findOrders(
       const priceBN = createBigNumber(price);
       let typeOp = type;
 
-      const outcomeName = outcomesData[outcome].name;
+      const outcomeName = getOutcomeName(marketsData, outcomesData[outcome]);
```

Comment:
<img width="1063" alt="Screen Shot 2019-04-26 at 4 17 02 PM" src="https://user-images.githubusercontent.com/6775839/56840856-cbebcc80-683e-11e9-88ff-80030d301bc4.png">
got this here

Reply:
I think that occurs when outcomesData hasn't been loaded. I'll put in a guard